frontend/services/gui/GUI_Service.cpp

```diff
@@ -188,7 +191,7 @@ void GUI_Service::digestChangedRequestedResources() {
 
 void GUI_Service::resetProvidedResources() {
 
-    // nothing to do here.
+    this->m_providedResource.request_gui_state = nullptr;
```

Comment:
Restting resources is intended to clean up resource state after a frame, before new resource state is written in the next frame. For exammple, keyboars events are deleted from event queues during resource reset and before updating the queues with new inputs. After a reset resources should be in a valid state. `request_gui_state` is not a resource that needs resetting.

Reply:
Jepp ok, in fact I had some headache about that ;-) Fixed in commit 00fe89a